src/spikeinterface/core/analyzer_extension_core.py

```diff
@@ -380,7 +382,12 @@ def _set_params(self, ms_before: float = 1.0, ms_after: float = 2.0, operators=N
         assert isinstance(operators, list)
         for operator in operators:
             if isinstance(operator, str):
-                assert operator in ("average", "std", "median", "mad")
+                if operator not in ("average", "std", "median", "mad"):
```

Comment:
I was thinking about this earlier, do you think it is worth adding the type  : `"average" | "std" | "median" | "mad" | None` to the `_set_params ` signature here? I do not think this is documented anywhere except for inside the function body